api/database/migrations/2015_07_29_233138_create_social_logins_table.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use App\Models\User;
+use Bosnadev\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSocialLoginsTable extends Migration
+{
+    const MODEL = 'App\Models\SocialLogin';
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $modelClass = static::MODEL;
+        Schema::create($modelClass::getTableName(), function (Blueprint $table) use ($modelClass) {
+            $table->uuid('social_login_id');
+
+            $table->uuid('user_id');
+            $table->string('provider');
+            $table->text('token');
+            $table->timestamps();
+
+            $table->primary('social_login_id');
```

Comment:
This table should also have a composite unique key of `user_id`, and `provider`. It might not actually need a surrogate `social_login_id` key as for example a delete request to remove a social login method could be `DELETE /user/{userId}/social-logins/facebook`